runtime/src/accounts_db.rs

```diff
@@ -165,6 +166,21 @@ impl AccountStorageEntry {
     }
 }
 
+pub struct ParThreadPool {
+    pub pool: ThreadPool,
+}
+
+impl Default for ParThreadPool {
+    fn default() -> Self {
+        ParThreadPool {
+            pool: rayon::ThreadPoolBuilder::new()
+                .num_threads(16)
```

Comment:
Default should probably be just 1 thread. To prevent unnecessarily large thread counts. For specialized use cases we should manually specify how many threads to use ?

Reply:
👍